tests/python/unittest/test_kvstore.py

```diff
@@ -283,6 +283,26 @@ def test_get_type():
     kvtype = 'local_allreduce_cpu'
     kv = mx.kv.create(kvtype)
     assert kv.type == kvtype
+    kvtype = 'teststore'
+    kv = mx.kv.create(kvtype)
+    assert kv.type == kvtype
+
+@with_seed()
+def test_custom_store():
```

Comment:
Should this method go to test_kvstore_custom.py ?

Reply:
i can do that